app/javascript/src/components/menus/activated_menu_activator.js

```diff
@@ -64,10 +68,12 @@ class ActivatedMenuActivator {
   #createNode() {
       const $node = $(
         createElement("button", 
```

Comment:
Isn't this change counter-productive to Accessibility in that you're replacing (presumably) useful text with nothing?

Reply:
The accessible text is exactly the same - the activator_textis what will be used as an accessible label and read out by screen readers.